scripts/build.py

```diff
@@ -152,12 +175,19 @@ def get_cache_slug():
         BUILD_DIR, 'third_party', 'generated')
     build_minified_third_party_libs(THIRD_PARTY_GENERATED_OUT_DIR)
 
-    # Process extensions, copy it to build/[cache_slug]/extensions
+    # Minify extension static resources, copy it to
+    # build/[cache_slug]/extensions
     EXTENSIONS_SRC_DIR = os.path.join('extensions', '')
     EXTENSIONS_OUT_DIR = os.path.join(BUILD_DIR, 'extensions', '')
-    copy_files_source_to_target(EXTENSIONS_SRC_DIR, EXTENSIONS_OUT_DIR)
-
-    _build_files()
+    # Certain files' syntax become incorrect after minification and hence
```

Comment:
Ah yes, I see, thanks!

Could you please file an issue saying that the files in this directory don't get minified correctly, with instructions explaining what command to use in order to attempt to minify them, and paste the error message that results in the issue? Then /cc @jacobdavis11 -- he created this interaction, so perhaps he can help with figuring out what's going on. That way, we don't lose track of this.

Thanks!